docs/releasenotes/11.2.0.rst

```diff
@@ -44,6 +44,18 @@ TODO
 API Additions
 =============
 
+"justify" multiline text alignment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In addition to "left", "center" and "right", multiline text can also be aligned using
+"justify"::
```

Comment:
```suggestion
``"justify"`` multiline text alignment
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

In addition to ``"left"``, ``"center"`` and ``"right"``, multiline text can also be aligned using
``"justify"``::
```
And maybe link to the module or methods reference?

https://pillow--8721.org.readthedocs.build/en/8721/reference/ImageDraw.html